NetCore/LoadResizeSave.cs

```diff
@@ -25,17 +26,22 @@ public class LoadResizeSave
         private const string ImageSharp = nameof(ImageSharp);
         private const string SystemDrawing = nameof(SystemDrawing);
         private const string MagickNET = nameof(MagickNET);
+        private const string NetVips = nameof(NetVips);
         private const string FreeImage = nameof(FreeImage);
         private const string MagicScaler = nameof(MagicScaler);
         private const string SkiaSharpCanvas = nameof(SkiaSharpCanvas);
         private const string SkiaSharpBitmap = nameof(SkiaSharpBitmap);
 
         // Set the quality for ImagSharp
-        private static readonly JpegEncoder imageSharpJpegEncoder = new JpegEncoder { Quality = Quality };
-        private static readonly ImageCodecInfo systemDrawingJpegCodec = ImageCodecInfo.GetImageEncoders().First(codec => codec.FormatID == ImageFormat.Jpeg.Guid);
+        private readonly JpegEncoder imageSharpJpegEncoder = new JpegEncoder { Quality = Quality };
+        private readonly ImageCodecInfo systemDrawingJpegCodec =
+            ImageCodecInfo.GetImageEncoders().First(codec => codec.FormatID == ImageFormat.Jpeg.Guid);
+
+        public readonly IEnumerable<string> images;
```

Comment:
If you're going to make this public, please capitalize the first letter and make it a get-only property instead of a readonly field.

Reply:
Oops, I should have seen that. It should be fixed now, thanks!